src/widgets.tsx

```diff
@@ -534,7 +640,373 @@ export const Dropdown_Select = (props:{validationLabel?:string, setSelected:((va
+                                />}
+                        </>
+
+                        )}
+                        name={field.field}
+                        key={field.field}
+                    />
+                    );
+                    break;
+                case InputType.SELECT_LIST:
+                    if (field instanceof InputSelectionField) {
+                        var selectListData:SelectListItem[] = [];
+                        for (var i=0; i<field.selectOptionList.length; i++) {
+                            selectListData.push({key: i, value: field.selectOptionList[i]})
+                        }
+
+                        const getSelectListDefaultValue = (selectListValue:string | undefined) => {
+                            if (selectListValue !== undefined) {
+                                for (var i=0; i<selectListData.length; i++) {
+                                    if (selectListData[i].value == selectListValue) return selectListData[i];
+                                }
+                            }
+                            return undefined;
+                        }
+
+                        return (
+                            <Controller 
+                                control={control}
+                                rules={{
+                                required: field.required,
+                                }}
+                                render={({ field: {onChange, onBlur, value}}) => (
+                                <>
+                                    {fieldValueIsString(field.type, value) &&                                
+                                    <Dropdown_Select
+                                        setSelected={(val:string) => onChange(val)}
+                                        data={selectListData}
+                                        placeholder={field.title}
+                                        validationLabel={(errors[field.field] && field.validationMessage) || undefined}
+                                        validationStyle={(errors[field.field] && {color: COLORS.primary, maxWidth: 250, alignSelf: "center"}) || undefined}
+                                        boxStyle={(errors[field.field] && {borderColor: COLORS.primary}) || {borderColor: COLORS.accent}}
+                                        defaultOption={getSelectListDefaultValue(value)}
+                                    />} 
+                                </>
+ 
+                                )}
+                                name={field.field}
+                                key={field.field}
+                            />
+                        );
+                    }
+                    else return (<></>)
+                    break;
+                case InputType.DATE:
+                    const minAge:Date = getDOBMaxDate(RoleEnum.STUDENT)
```

Comment:
We have `InputType.Date` that are not `dateOfBirth`.  So above this switch statement:
```
if(field.field === 'dateOfBirth`) {
     const minAge:Date = getDOBMaxDate(getField('userRole') || RoleEnum.STUDENT);
     const maxAge:Date = getDOBMinDate(getField('userRole') || RoleEnum.STUDENT);
     return <DateInput min={minAge} max={maxAge} />;
} else ...

Reply:
Validation is performed at the Controller level. Because of that, i'd rather not send data for validation at the widget level and perform validation twice. See new code in commit

Reply:
That'll work for now.  Prayer Request `expirationDate` I beleve needs to be in the future.  The config file has a `minValue`.  But that only reflects the last build date.  So we'll need to figure something out.